package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "quorum-dev-quickstart",
-  "version": "0.0.4",
+  "version": "0.0.5",
```

Comment:
This should be `0.0.5-beta.0` - we should only commit changes w/ a non-beta version on release commits.

Honestly this was my bad for not including a second "next version beta bump" commit when I made the last release.

Reply:
If you want, maybe just revert this change altogether and I'll make a commit to `master` to handle this, and you can rebase against that?

Reply:
Sure, lets do it. Let me know when I can rebase